packages/suite-desktop-core/src/config.ts

```diff
@@ -25,6 +25,9 @@ export const allowedDomains = [
     'trezor-cardano-mainnet.blockfrost.io',
     'trezor-cardano-preview.blockfrost.io',
     'blockfrost.dev',
+    'eth-goerli.public.blastapi.io',
+    'ethereum-holesky.publicnode.com',
+    'mainnet.infura.io/v3/f583d4f04d384b9e8c59a7ff1c9f68f1',
```

Comment:
```suggestion
    'mainnet.infura.io',
```

Reply:
_(As we discussed, those backends are temporary. I raised a question if we can provide RPC using our blockbook, if needed)_

Reply:
https://github.com/everstake/trezor-suite/pull/7/commits/6f35bf0e58d70245d04b4ced2b5d2af3d7bf2558